mico-core/src/test/java/io/github/ust/mico/core/ApplicationControllerTests.java

```diff
@@ -169,7 +175,7 @@ public void getApplicationByShortName() throws Exception {
 
     @Test
     public void getApplicationByShortNameWithTrailingSlash() throws Exception {
-        given(applicationRepository.findByShortName(SHORT_NAME)).willReturn(Collections.singletonList(new MicoApplication().setShortName(SHORT_NAME).setVersion(VERSION)));
+        given(applicationRepository.findByShortName(SHORT_NAME)).willReturn(CollectionUtils.listOf(new MicoApplication().setShortName(SHORT_NAME).setVersion(VERSION)));
```

Comment:
Is there a reason to change Collections.singletonList to CollectionUtils.listOf ?

Reply:
I changed it only to be consistent. We use `CollectionUtils.listOf` at all other places to also support the creation of a list with multiple elements. I'm aware of that it is in comparison with `Collections.singletonList` not immutable, but that should be no problem.